tests/examplefiles/nim/example.nim.output

```diff
@@ -907,21 +905,19 @@
 'askSave'     Name
 ')'           Punctuation
 ')'           Punctuation
-'\n      '    Text
+'\n      '    Text.Whitespace
 'case'        Keyword
-' '           Text
+' '           Text.Whitespace
 'resp'        Name
-'\n      '    Text
-'of'          Keyword
-' '           Text
-'RESPONSE_ACCEPT' Name
-':'           Punctuation
-'\n        '  Text
+'\n      of ' Keyword
```

Comment:
Is this really desired to match the whitespace as part of the keyword?

Reply:
Fixed. Please review, thanks.